r-package/grf/R/regression_forest.R

```diff
@@ -217,14 +217,18 @@ get_params <- function(X, draw) {
 }
 
 #' Predict with a regression forest
-#' 
+#'
 #' Gets estimates of E[Y|X=x] using a trained regression forest.
 #'
 #' @param object The trained forest.
 #' @param newdata Points at which predictions should be made. If NULL,
 #'                makes out-of-bag predictions on the training set instead
 #'                (i.e., provides predictions at Xi using only trees that did
 #'                not use the i-th training example).
+#' @param local.linear Optional local linear prediction correction
```

Comment:
Might be good to elaborate a tiny bit more on the algorithm? You can also mention that setting this to true will cause quite a slow down in prediction, and that this is a beta feature that we expect to make improvements to. Note these comments are also typically full sentences.